WTWSMS/localisation/LI_relf_localisation.csv

```diff
@@ -1,5 +1,5 @@
 tolerant_ruler;Tolerant Ruler;;;;;;;;;;;;;x
-tolerant_ruler_desc;This ruler has approved crown laws granting freedom of cult under their realm. The tolerant regime allows them to treat fairly better with some of their subjects, but alienate them the sympathies of the more zealous members of their own claimed religion.;;;;;;;;;;;;;x
+tolerant_ruler_desc;This ruler has approved crown laws granting freedom of cult under their realm. In their realm, heretics and heathens are allowed to freely follow their customs and rites, without fear of persecution. The tolerant regime brings great peace and loyalty among heathen subjects, but alienates the sympathies of the more zealous members of the ruler's own religion.;;;;;;;;;;;;;x
```

Comment:
That's what mine says. "Freely follow their customs and rites without fear of persecution" means that they won't be attacked for their religion. Thus, they are treated fairly.